darts/darts.py

```diff
@@ -1,2 +1,14 @@
+x = int(input('Please enter the x coordinate:'))
+y = int(input('Please enter the y coordinate:'))
 def score(x, y):
-    pass
```

Comment:
I think we need to return the score as an integer instead of as a string...no biggie